include/btllib/MIBloomFilter.hpp

```diff
@@ -0,0 +1,915 @@
+/*
+ * MIBloomFilter.hpp
+ *
+ * Agnostic of hash function used -> cannot call contains without an array of
+ * hash values
+ *
+ *  Created on: Jan 14, 2016
+ *      Author: cjustin
+ */
+
+#ifndef MIBLOOMFILTER_HPP_
+#define MIBLOOMFILTER_HPP_
+
+#include <algorithm> // std::random_shuffle
+#include <cassert>
+#include <cmath>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <limits>
+#include <omp.h>
+#include <sdsl/bit_vector_il.hpp>
+#include <sdsl/rank_support.hpp>
+#include <string>
+#include <sys/stat.h>
+#include <vector>
+
+namespace btllib {
+using std::cerr;
+using std::endl;
+using std::ios;
+using std::ofstream;
+using std::pair;
+using std::string;
+using std::vector;
+
+template<typename T>
+class MIBloomFilter
+{
+public:
+  static const T MASK = 1 << (sizeof(T) * 8 - 1);
+  static const T ANTI_MASK = (T)~MASK;
+
+  static const T STRAND = 1 << (sizeof(T) * 8 - 2);
+  static const T ANTI_STRAND = (T)~STRAND;
+
+  static const T ID_MASK = ANTI_STRAND & ANTI_MASK;
+
+  static const unsigned BLOCKSIZE = 512;
+  // static methods
+  /*
+   * Parses spaced seed string (string consisting of 1s and 0s) to vector
+   */
+  static inline vector<vector<unsigned>> parse_seed_string(
+    const vector<string>& spaced_seeds)
```

Comment:
Not sure why this one gives unused parameter. I'd try debugging it by removing parts of the function code and running CI again until you pinpoint the moment when it starts giving the error.